pkg/plugin/examples/python/pyraw.yml

```diff
@@ -0,0 +1,29 @@
+# example plugin config
+name: Hawwwwt Tagger (Raw Python edition)
+description: Python Hawwwwt tagging utility (using raw interface).
+version: 1.0
+url: http://www.github.com/stashapp/stash
+exec:
+  - python
```

Comment:
users that have python2 also present need to edit `python` to `python3` for the delete tag to work

Reply:
Since it's just a proof of concept example, I'll leave it as is.